meson.build

```diff
@@ -1715,6 +1715,8 @@ conf.set10('ENABLE_PAM_HOME', have)
 have = get_option('oomd')
 conf.set10('ENABLE_OOMD', have)
 
+conf.set10('ENABLE_VARLINK_NOTIFICATIONS', get_option('dns-varlink-notifications'))
```

Comment:
I know @bluca requested a config option, but I don't think that's useful. This should be settable through local configuration. The amount of code is tiny, so adding a configuration option for this is not useful.

Below the function that sends this out checks if there are any clients. And clients need to be privileged to connect, so that should be enough.

Reply:
Yeah, but if root privileges are required to actually use this, I don't think a separate option to enable it is useful at all. (Re-reading my comment above, it's not very clear. I don't think we need the option at all.)

Reply:
The potential for privacy shenanigans complaints is absolutely massive here, i really think it needs to be off by default

Reply:
Well, if you have root privs, then you can just enable it. Or if it's not enabled in the installed version, install a different version. What would the point of adding a switch that is under control of the same user be?

Reply:
Auditing and perception.

Changing a config file and restarting are much more explicit and auditable actions than simply starting to listen on a socket. In other words, if you want to know if anything on your system is snooping on your dns requests, with an opt-in all you need to check is the status of the config file. If it's always enabled, then you need to constantly monitor access to the socket.

Also having it default-enabled I'm pretty sure is going to be badly perceived and get us bad publicity... in general with things that expose end-user data, erring on the cautious side seems prudent to me. I imagine it's going to be useful for debugging purposes anyway, so having a runtime toggle doesn't seem so bad to me?

Reply:
the config file already exists and is already parsed, you can check how it's  handled in `src/resolve/resolved-conf.c`

Reply:
I've updated this to be a runtime toggle in the latest patch, and removed the build-time option. The setting is called "Monitor=" and defaults to "no"